pkg/agent/controller/traceflow/packetin.go

```diff
@@ -274,3 +293,15 @@ func getNetworkPolicyObservation(tableID uint8, ingress bool) *opsv1alpha1.Obser
 	}
 	return ob
 }
+
+func getInfoInCtIPv6DstField(matchers *ofctrl.Matchers) (string, error) {
```

Comment:
Probably: Ct -> CT
How about getCTIPv6DstValue()? Guess you need to rename the funcs for other fields too, if you rename this one.

Reply:
Changed to "getInfoInCTDst" for both IP and IPv6 functions.